src/homework19/GaragePage/GaragePage.js

```diff
@@ -0,0 +1,20 @@
+import BasePage from "../BasePage.js";
+import {SideBarMenu} from "./components/SideBarMenu.js";
+import Header from "../../components/Header.js";
+
+
+export default class GaragePage extends BasePage{
+    constructor(page) {
+        super(page, "/panel/garage");
+        this.titleGaragePage =  page.locator('h1', {hasText: 'Garage'});
+    };
+
+    async sideBarInstance() {
+        return new SideBarMenu(this._page);
+    }
```

Comment:
not this one

Reply:
fixed in https://github.com/HelensBadCode/AQA_playwright/pull/4/commits/6a148f556810e850b7c928855bcf90190e00fe0f